dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -580,12 +583,12 @@ def test_profile(self):
                 'bin_edges': np.array([2.5, 5.0, 7.5, 10.0, 12.5]),
             },
             quantiles={
-                0: 3.75,
-                1: 5.0,
-                2: 10.0
+                0: 4.375,  # 75% between 2.5 and 5 (histogram based)
```

Comment:
what does 75% mean? Can you elaborate the comment?

Reply:
(5-2.5)*.75 + 2.5, trying to keep it from being multiline, do you want me to put that instead?

Reply:
imo, you should put a comment explaining why do you have 75% from histogram, from that it's clear how we get number 4.375.

Reply:
adding, thanks!